src/hooks/useSubaccount.tsx

```diff
@@ -346,11 +349,12 @@ export const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: Lo
 
     try {
       await getFaucetFunds({ dydxAddress, subaccountNumber });
+      await getNativeTokens({ dydxAddress });
```

Comment:
can they be done in parallel? can we do something like
`await Promise.all([getFaucetFunds(...), getNativeTokens(...)])`

Reply:
Ah yeah! Will add to promise.all